### DISCUSSION TITLE
Return to Manga Overview when on Last Page of Last Chapter

### DISCUSSION THREAD

**Discussion by Kinasho** [xgi/houdoku#140] [Feature Requests]
If you use the arrow keys to read manga, on the last page of the last chapter, when you use the arrow keys or mouse to seek forward again, nothing will occur. 

I would like for there to be a feature whereby if you click or use the arrow key on the last page of the last chapter of a manga, it wouldn't just do nothing but instead it would return the user to the manga overview page.

![image](https://user-images.githubusercontent.com/106453771/170845527-9783bd32-65cc-4ecb-98b2-2302a70c5903.png)

**Comment by xgi** [comment]
I don't think I'm in favor of this, I think it would be a bit confusing and unwelcome if someone didn't realize they were on the last page and got "kicked out" after trying to go to the next page. There's still keybinds they can use to exit the reader, if they want to (`ESC/backspace`).

**Reply by ScruffiX** [reply]
Hm idk, personally I found it more confusion/irritating when I clicked on the last page and nothing happened. I'd agree with Kinasho's suggestion.

**Reply by Kinasho** [reply]
I can see where both of  you are coming from. How about instead, if there's a warning or something which informs the reader that they've reached the last page, whether that be a text popup, or a next page which says "No More Pages" or something. Of course, these are just things I thought of while typing this, but if there's something to inform the reader that they've reached the end, it wouldn't be frustrating. (Of course, such a thing would only appear if the user were to try moving on to the next page while on the last page. You wouldn't want to obstruct their view of the last page at all)

**Reply by xgi** [reply]
Adding a prompt is definitely a good compromise imo. Theoretically it could have a "don't show this again" option as well, although I may not include that initially.

**Reply by Kinasho** [reply]
Thanks for considering it. Sounds good
